tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -29,12 +29,23 @@ user signs in as analyst1
 
     user waits until h1 is visible  Dashboard
     user waits until page contains title caption  Welcome Analyst1
+    
     user waits until page contains element   id:publicationsReleases-themeTopic-themeId   180
 
     user checks breadcrumb count should be  2
     user checks nth breadcrumb contains  1   Home
     user checks nth breadcrumb contains  2   Administrator dashboard
 
+user signs in as analyst1 as a publication owner 
```

Comment:
An analyst with publication owner access will not be able to see some of the stuff that `user signs in as analyst1` asserts. Would it be best to move this into the test itself rather than a kw in `admin-common` ? 

```
  user waits until page contains element   id:publicationsReleases-themeTopic-themeId   180

    user checks breadcrumb count should be  2
    user checks nth breadcrumb contains  1   Home
    user checks nth breadcrumb contains  2   Administrator dashboard
```